i18n/core.json

```diff
@@ -7,8 +7,35 @@
   "UIKit.CreatableSelectInput.createLabel": "Create \"{inputValue}\"",
   "UIKit.CreatableSelectInput.noOptionsMessageWithInputValue": "No options",
   "UIKit.CreatableSelectInput.noOptionsMessageWithoutInputValue": "No options",
-  "UIKit.DateInput.labelRange": "to",
-  "UIKit.DateTimeInput.labelRange": "to",
+  "UIKit.DateInput.lastDay": "[Yesterday], Do MMM YYYY",
```

Comment:
this is where it might make sense to add a US and UK locale 🤔

Reply:
Might be an idea for another PR :)